plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -111,7 +111,7 @@
     private static final String DEFAULT_SOURCE_ID = "default";
 
     /** First version with new parent POM. */
-    public static final String JENKINS_CORE_FILE_REGEX = "WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:-(?i)([a-z]+)(-)?([0-9.]+)?)?(?:-SNAPSHOT)?)[.]jar";
+    public static final String JENKINS_CORE_FILE_REGEX = "WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9a-f.]+)*(?:-(?i)([a-z]+)(-)?([0-9a-f.]+)?)?(?:-(?i)([a-z]+)(-)?([0-9a-f.]+)?)?(?:-SNAPSHOT)?)[.]jar";
```

Comment:
Why so specific? Why do we not just match any `jenkins-core-*.jar`?

Reply:
I agree with @jglick here, it could make sense in the past to be specific but at this point, I am afraid trying to cover all cases will only make this too much complicated for no real reason. That would be different in the case of the PCT being offered under a Saas model (which I believe happened in the past) but now every run of the PCT I could think of is local so you do not really need to protect yourself against "funny" versions of core